lib/client.js

```diff
@@ -147,6 +147,24 @@ Freshdesk.prototype = Object.create({
 		makeRequest('GET', this._auth, `${this.baseUrl}/api/v2/contact_fields`, null, null, cb)
 	},
 
+	//Agents
+
```

Comment:
Could you add a JSDoc here ([example](https://github.com/arjunkomath/node-freshdesk-api/blob/master/lib/client.js#L46))? For all four new methods (actually, there should be 5 new methods).

Don't forget to add **filter** example for `listAllAgents`